app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/DRecordDetailsFragment.kt

```diff
@@ -1,44 +1,79 @@
 package com.github.factotum_sdp.factotum.ui.roadbook
 
 import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import android.widget.EditText
+import android.view.*
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
 import com.github.factotum_sdp.factotum.R
 import com.github.factotum_sdp.factotum.data.DestinationRecord
+import androidx.appcompat.widget.Toolbar
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import androidx.viewpager2.widget.ViewPager2
+import com.github.factotum_sdp.factotum.ui.directory.DirectoryFragment
+import com.github.factotum_sdp.factotum.ui.maps.RouteFragment
+import com.github.factotum_sdp.factotum.ui.picture.PictureFragment
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
 
 class DRecordDetailsFragment: Fragment() {
 
     private val rbViewModel: RoadBookViewModel by activityViewModels()
+    private lateinit var rec: DestinationRecord
+    private lateinit var viewPager: ViewPager2
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.fragment_drecord_details, container, false)
-        val arg = arguments?.getString(RoadBookFragment.DEST_ID_NAV_ARG_KEY) ?: "UNKNOWN"
+
+        val destID = arguments?.getString(RoadBookFragment.DEST_ID_NAV_ARG_KEY) ?: "UNKNOWN"
         rbViewModel.recordsListState.value?.let {
-            val rec = it.first { d -> d.destID == arg }
-            setEditTexts(rec, view)
+            rec = it.first { d -> d.destID == destID }
         }
 
+        val toolbar = requireActivity().findViewById<Toolbar>(R.id.toolbar)
+        toolbar.setTitle("Dest. Record : $destID")
+
+        viewPager = view.findViewById(R.id.viewPager)
+        val adapter = DetailsFragmentsAdapter(this)
+        adapter.addFragment(DRecordInfoFragment(rec))
+        adapter.addFragment(RouteFragment())
+        adapter.addFragment(DirectoryFragment()) // To be replaced with ContactDetailsFragment()
```

Comment:
Should we add a TODO?

Reply:
I think it doesn't need. It's clearly obvious as there is a designated task in the current Sprint for that. And it's not a hidden code place. The task programmer will be forced to arrive there and see that comment.